src/components/CryptoSelectForm.js

```diff
@@ -5,15 +5,18 @@
 		---The main reason why there is still this component and the IntlSelectForm is
 		because I couldn't get the crypto.name to render conditionally while being mapped.
 */
-
+/*
+	I added the animation and delay styles which are passed in via props from
+	the CryptoTraq component.
+ */
```

Comment:
I added the animation and delay styles which are passed in via props from the `CryptoTraq` component.
 When the app loads, the two `select` elements glow in succession to help guide the user on how to use the app.